lib/js/importers/AllInOneImporter.ts

```diff
@@ -0,0 +1,189 @@
+const fsT = require('fs');
+const axios = require("axios");
+const tokensFilesConfig = JSON.parse(
+	fsT.readFileSync('lib/js/importers/configs/allInOneConfig.json', 'utf8')
+);
+
+interface IResultJsonObject {
+	id: string;
+	label: string;
+	value: string;
+	weight: number;
+}
+
+interface configFileObject {
+	destination: string,
+	destinationJson: string
+}
+
+interface ITokenJsonObject {
+	id: string;
+	label: string;
+	value: string;
+}
+
+const ImportColorsRaw = (
+	binValues: configFileObject,
+	jsonSource: { data }
+) => {
+	let hexToCssVariable: Array<Object> = [];
+	try {
+		let result: Array<string> = [];
+		let resultJson = {};
+
+		jsonSource.data.colors.forEach(obj => {
+			const patternIsRawColor = /RAW\/|theme/i;
+			const patternDeprecated = /deprecated|Pattern/i;
+			const patternTheme = /theme/i;
+			if (obj.name.match(patternIsRawColor) && obj.name.match(patternDeprecated) === null) {
+				const nameSplit = obj.name.split('/');
+				let colorName = (nameSplit[2] === undefined) ? nameSplit[1] : nameSplit[2];
+
+				obj.values.hex = obj.values.hex.replace(/FFFFFF/gi, function(){
+					return 'fff';
+				});
+
+				const colorPrefix = obj.name.match(patternTheme) ? '--' : '--raw-';
+
+				if (hexToCssVariable[obj.values.hex] === undefined) {
+					hexToCssVariable[obj.values.hex] = colorPrefix + colorName;
+				}
+
+				result.push(colorPrefix + colorName + ': ' + obj.values.hex + ';');
+				result.push(colorPrefix + colorName + '-rgb' + ': ' + hexToRgb(obj.values.hex) + ';');
+
+				/** JSON object structure */
+				colorName = colorName.replace(/--/i, '').replace(/raw-/i, '');
+				const colorNameSplitted = colorName.split('-');
+				const category = (colorNameSplitted[1] === undefined) ? 'default' : colorNameSplitted[0];
+				const resultJsonObject: IResultJsonObject = {
+					id: binValues.destination + '_' + colorName,
+					label: colorName,
+					value: obj.values.hex,
+					weight: colorNameSplitted[2] ? parseInt(colorNameSplitted[2]) : parseInt(colorNameSplitted[1])
```

Comment:
I believe `colorNameSplitted[2]` is not defined in current bins

Reply:
Do we use `weight` after we sort values? Maybe we should remove it?

Reply:
I did it. I'm sure there is smarter way, but I spend too much time to try optimise it. Maybe you have some idea. I mean casting object to another object, I was sure it will work but it didn't:
`const resultFileJsonObject = <IResultFileJsonObject> color`

Reply:
`weight` is not used now. Maybe we should remove it?

Reply:
Removed.